components/constellation/constellation.rs

```diff
@@ -1181,6 +1195,13 @@ impl<Message, LTF, STF> Constellation<Message, LTF, STF>
             }
         }
 
+        if let Some(chan) = self.webvr_thread.as_ref() {
+            debug!("Exiting WebVR thread.");
```

Comment:
I included it inside the `if` because WebVR can be disabled and it didn't make sense to show that message. I checked and the same pattern is used in ""Exiting devtools" too ;)